status/flowcells.py

```diff
@@ -105,6 +100,11 @@ def list_flowcells(self, all=False):
 
             # Lanes were previously in the wrong order
             row.value["lane_info"] = OrderedDict(sorted(row.value["lane_info"].items()))
+            latest_note = LatestRunningNoteHandler.get_latest_running_note(
```

Comment:
I don't really like that we're making one http request to couch for each flowcell in the list. It should be possible to make a single request and then fetch the needed individual rows from that request?